tests/e2e/before-deployment-test-cleanup.sh

```diff
@@ -13,6 +13,8 @@
 #     See the License for the specific language governing permissions and
 #     limitations under the License.
 
+helm -n dynatrace ls --all --short | grep dynatrace-gcp-function | xargs -L1 helm -n dynatrace uninstall
```

Comment:
Please add --wait 
https://helm.sh/docs/helm/helm_uninstall/

Reply:
It occurred that --wait is not supported by uninstall command.